docs/quickstart.md

```diff
@@ -126,7 +126,7 @@ Dredd's [hooks](hooks.md) enable you to write some glue code in your favorite la
 For more complex example applications, please refer to:
 
 - [Express.js example application](http://github.com/apiaryio/dredd-example)
-- [Ruby on Rails example application](https://github.com/theodorton/dredd-test-rails)
+- [Ruby on Rails example application](https://github.com/apiaryio/dredd-test-rails)
```

Comment:
Ha, no problem! I assumed it's easier to fix the link than to maintain repo mirror :) Just wanted to know if the apiary repo is some sort of backup fork of the original repo or what. Not sure if it contains any differences from your repo, will have to check.